packages/docs/src/page-configs/getting-started/kitchensink/page-config.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiDocsBlock, BlockType } from '@/types/configTypes'
+import { shallowReactive } from 'vue'
+
+const importedConfigsContext = require.context(
+  '@/page-configs/ui-elements/',
+  true,
+  /page-config.ts$/,
+  'lazy',
+)
+
+export const config = shallowReactive<ApiDocsBlock[]>([])
+
+importedConfigsContext.keys().forEach(async (filename) => {
+  let importedConfig = null
+  try {
+    importedConfig = (await importedConfigsContext(filename)).default
```

Comment:
```suggestion
    importedConfig = (await importedConfigsContext(filename))?.default
```
If here is no way it will return null or undefined, ignore this. Or you rely on try-catch?